.gitignore

```diff
@@ -29,3 +29,4 @@ web.*.config
 waca.sln.DotSettings.user
 .vs/*
 .arc*
+password.php
```

Comment:
Be careful about ignoring .php files in a PHP project :grinning:

I'd be tempted to add this it `.git/info/exclude` since it's specific to your exact configuration